packages/core/src/internal/clients/cache/AbstractCacheClient.ts

```diff
@@ -658,9 +670,9 @@ export abstract class AbstractCacheClient implements ICacheClient {
   }
 
   /**
-   * Associates the given key with the given value. If a value for the key is
-   * already present it is not replaced with the new value.
+   * Associates the given key with the given value if key is not already present in the cache.
    *
+   * @deprecated Use setIfAbsent instead.
```

Comment:
oh I see, you did it here, thanks. Probably a good idea to do it in the interface too.